node/internal/child_process.ts

```diff
@@ -411,6 +446,173 @@ function normalizeStdioOption(
   }
 }
 
+export function normalizeSpawnArguments(
+  file: string,
+  args: string[],
+  options: SpawnSyncOptions,
```

Comment:
FWIW, Node calls `normalizeSpawnArguments()` for both `spawn()` and `spawnSync()`. No need to change this now, but something to think about for the future.